musit-service/src/test/scala/no/uio/musit/healthcheck/ZabbixExecutorSpec.scala

```diff
@@ -75,12 +56,28 @@ class ZabbixExecutorSpec extends MusitSpec with BeforeAndAfterEach {
       val res = Json.parse(new FileInputStream(zibbixFile.ensureWritableFile()))
       (res \ "updated").as[Long] mustBe latestUpdated
     }
+
+    "write result over old file even when the state changes" in {
+      val folder     = createTempFolder()
+      val zibbixFile = ZabbixFile(folder, "musit-health.json")
+      val executor: ZabbixExecutor =
+        createExecutor(zibbixFile, new FlippingAvailableHealthCheck)
+
+      (1 to 3).foreach { n =>
```

Comment:
regardsless of how you try to pronounce `(1 to 3)` it sounds like one, two, three 😁